doc/tasks.rst

```diff
@@ -307,3 +307,19 @@ Luigi also does some metaclass logic so that
 if e.g. ``DailyReport(datetime.date(2012, 5, 10))`` is instantiated twice in the code,
 it will in fact result in the same object.
 See :ref:`Parameter-instance-caching` for more info
+
+
+Atomicity of Operations on Targets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Atomicity is crucial when working with targets. In the following scenarios
+atomicity cannot be guaranteed.
+
+Moving of a :class:`~luigi.target.LocalTarget` across filesytems:
+
+.. code:: python
+
+    luigi.LocalTarget(path=src).move(dst)
```

Comment:
Wouldn't it be much more easy to put this in the docs for `LocalTarget.move()` docs?? You don't even need a code example. Just a warning saying to be aware of the risks when doing this across filesystems's.